### PR TITLE
Feat/placesBeenTableCss

### PR DESCRIPTION
**Pull request by ottonotto**

### CODE REVIEW

**Comment by ottonotto** [comment]
ready for review.

File: src/App.js [hunk 1]
```diff
@@ -29,9 +29,8 @@ const App = () => {
                 <Route exact path="/" element={<Home />} />
                 <Route path="/PlacesIWantToGo" element={<PlacesIWantToGo />} />
                 <Route path="/PlacesIHaveBeen" element={<PlacesIHaveBeen />} />
-                <Route path="/Logout" element={<Logout />} />
             </Routes>
-            <Logout />
+            <Navbar />
```
Comment (shams-ali) [inline src/App.js:33]: you want to layout your code according to the view...you should navbar above routes not below

File: src/components/pages/PlacesIHaveBeen/index.jsx [hunk 2]
```diff
@@ -1,11 +1,132 @@
...
-    <>
+    <div>
         <img src={backgroundImage} className="background" alt="Ocean Park" />
         <h1 className="header">Places I Have Been!</h1>
-    </>
+        <div>
+            <table className="table">
+                <thead>
+                    <tr>
+                        <th>Places I Have Visted</th>
```
Comment (shams-ali) [inline src/components/pages/PlacesIHaveBeen/index.jsx:39]: this should be in an array and then looped over

File: src/components/pages/PlacesIHaveBeen/index.jsx [hunk 3]
```diff
@@ -1,11 +1,132 @@
...
+                        <th>What I Did</th>
+                        <th>When I Went</th>
+                        <th>Who Went?</th>
+                        <th>Rating</th>
+                    </tr>
+                </thead>
+                {data.map(val => (
+                    <tbody>
+                        <tr>
+                            <td>{val.places}</td>
```
Comment (shams-ali) [inline src/components/pages/PlacesIHaveBeen/index.jsx:50]: deconstruct

File: src/App.js (hunk 1 above)
Comment (ottonotto) [reply src/App.js:33]: fixed

**Review by ottonotto — COMMENTED** [review]
ready for review

**Pull request closed without merging**